samples/sensor/sensor_shell/sample.yaml

```diff
@@ -5,6 +5,7 @@ tests:
     integration_platforms:
       - frdm_k64f
     filter: ( CONFIG_UART_CONSOLE and CONFIG_SERIAL_SUPPORT_INTERRUPT )
+    platform_exclude: nsim_hs6x_smp
```

Comment:
This is probably not great but I'm not sure how to resolve the build issue with this board.

Reply:
Hi @nixward,

Please share some details about the reason you need disable `nsim_hs6x_smp` platform. Ae we bumping int build time / runtime issues?

If so could you please share the logs?

Thanks!

Reply:
Sure thing. This is one of the past failing CI runs with the `nsim_hs6x_smp` platform:
https://github.com/zephyrproject-rtos/zephyr/runs/15751731400

Reply:
Hi @nixward,

I expect that with [this fix ](https://github.com/zephyrproject-rtos/zephyr/pull/61903) we no longer need tins platform_exclude.

May I ask you to rebase?
Thanks!

Reply:
Rebased